test/integration/options.spec.js

```diff
@@ -180,4 +183,28 @@ describe('options', function () {
       });
     });
   });
+
+  describe('--help', function () {
+    it('works despite the presence of mocha.opts', function (done) {
+      var output = '';
```

Comment:
This initialization is redundant since an implicit casting is being done on line 195.